src/components/VolumeSlider/VolumeSlider.js

```diff
@@ -10,7 +10,60 @@ class VolumeSlider extends Component {
     onVolumeChanged: PropTypes.func.isRequired,
   };
   render() {
-    return <div>VolumeSlider</div>;
+    const { volume, onVolumeChanged } = this.props;
+    const width = 294;
+    const maxVolume = 100;
+    const currentVolumeWidth = (volume / maxVolume) * width;
```

Comment:
Maybe need a `defaultProps` value for volume to avoid the `NaN` bug

Reply:
Added default volume in https://github.com/digirati-co-uk/timeliner/pull/57/commits/948e9f35511fd64e40e1616af74146ea67989444